openedx/features/enterprise_support/api.py

```diff
@@ -569,6 +570,45 @@ def get_enterprise_learner_data(user):
             return enterprise_learner_data['results']
 
 
+@enterprise_is_enabled()
+def get_enterprise_learner_portal_enabled_message(request):
+    """
+    Returns message to be displayed in dashboard if the user is linked to an Enterprise with the Learner Portal enabled.
+
+    Note: request.session['enterprise_customer'] will be used in case the user is linked to
+        multiple Enterprises. Otherwise, it won't exist and the Enterprise Learner data
+        will be used. If that doesn't exist return None.
+
+    Args:
+        request: request made to the LMS dashboard
+    """
+    if request.session['enterprise_customer']:
```

Comment:
@chavesj: I think this line is the source of our new errors with the latest release.